web_drop_target/static/src/js/web_drop_target.js

```diff
@@ -27,67 +28,62 @@ odoo.define('web_drop_target', function(require) {
         },
 
         _on_drop: function(e) {
-            var drop_item = this._get_drop_item(e);
-            if(!drop_item) {
+            var drop_items = this._get_drop_items(e);
+            if(!drop_items) {
```

Comment:
pls rebase and include this fix https://github.com/OCA/web/pull/1209

Reply:
Done